### DISCUSSION TITLE
⚠️YARP devel branch freeze

### DISCUSSION THREAD

**Discussion by drdanz** [robotology/community#277] [Announcements]
### TL;DR YARP DEVEL BRANCH FREEZE, DO NOT USE YARP DEVEL UNTIL FURTHER NOTICE

Hello all,

we are going to do some very important changes on the YARP repository, therefore all the PRs containing new features will have to wait until this is over, and will have to be rebased. Any unauthorized commit on YARP `devel` branch will be removed (using force push) therefore do not push anything on the `devel` branch, since you might lose your work.

PRs on the `master` branch (bug fixes) are still allowed, but please don't merge the `master` branch into `devel` (CC @pattacini).

Due to the undergoing changes, the `devel` branch might be temporarily broken, or missing some important feature, therefore I strongly suggest to **switch all robots and all your clones to YARP `master`**. Since we released YARP 2.3.72 a couple of days ago, `master` and `devel` are aligned, and new features are currently not allowed, therefore using `master` you are not missing any feature.
Please note that this is only a temporary issue, and that we are not planning to break the compatibility with previous versions.

I will add a comment and close this issue as soon as the freeze is over.

**Comment by drdanz** [comment]
robotology/yarp#1551 was merged, this means that robotology/yarp@3b2c4a3bccf4db0d926fc3c77e01ccb0d0b88a4e was the last "safe" commit if you are still using `devel`.

**Comment by drdanz** [comment]
All functionalities should be restored, and the repository is no longer frozen. Sorry if it took so long.
If you have open pull requests for yarp devel, please rebase them over the latest commit.
